ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables.md

```diff
@@ -14,13 +14,30 @@
   - IBGP (only with OSPF or ISIS variants in underlay)
 - Only summary network addresses need to be defined. IP addresses are then assigned to each node, based on its unique device id.
   - To view IP address allocation and consumption, a summary is provided in the auto-generated fabric documentation in Markdown and CSV format.
-- The variables should be applied to all devices in the fabric.
 
 *Only supported with core_interfaces data model.
 
-**Variables and Options:**
+## Flagging a Device as Not Deployed
+
+- It is possible to provision configurations for a complete topology but flag devices as undeployed using the host level variable `is_deployed: false`.
+
+  ```yaml
```

Comment:
```suggestion
```yaml
```

Reply:
I don't see end quotes **```** for this

Reply:
sorry I missed this before mergning. It renders correctly, so let's fix this in another PR touching this file (I have a few pending, so I will add it there)

Reply:
ah yeah copy paste issue I had them in my test .. I check the rendered file and it looked ok

however we should not remove the two spaces in front of ```yaml as we want the snippet to be indented under the bulletpoint.

Reply:
agreed. I have added the closing ``` to the trunk_groups PR.